sphinx/domains/cpp.py

```diff
@@ -3318,16 +3320,20 @@ def describe_signature(self, signode: TextElement, mode: str,
 
 
 class ASTClass(ASTBase):
-    def __init__(self, name: ASTNestedName, final: bool, bases: List[ASTBaseClass]) -> None:
+    def __init__(self, name: ASTNestedName, final: bool, bases: List[ASTBaseClass],
+                 attrs: Optional[List[ASTAttribute]] = None) -> None:
```

Comment:
```suggestion
                 attrs: List[ASTAttribute]) -> None:
```

Reply:
This is an internal class so it doesn't need to be backwards compatible, so I prefer to make it explicit that the AST has changed.